src/scripts/main.js

```diff
@@ -354,7 +354,42 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard');
 
-// write your code here
+function addRow(object) {
+  const row = document.createElement('tr');
+
+  const nameCell = document.createElement('td');
+
+  nameCell.textContent = object.name;
+  row.appendChild(nameCell);
+
+  const genderCell = document.createElement('td');
+
+  genderCell.textContent = object.sex;
+  row.appendChild(genderCell);
+
+  const bornCell = document.createElement('td');
+
+  bornCell.textContent = object.born;
+  row.appendChild(bornCell);
+
+  const diedCell = document.createElement('td');
+
+  diedCell.textContent = object.died;
+  row.appendChild(diedCell);
+
+  const ageCell = document.createElement('td');
+
+  ageCell.textContent = +object.died - +object.born;
+  row.appendChild(ageCell);
+
+  const centuryCell = document.createElement('td');
+
+  centuryCell.textContent = Math.ceil(object.died / 100);
+  row.appendChild(centuryCell);
+
+  table.appendChild(row);
+}
+
+people.forEach((person) => addRow(person));
```

Comment:
The `forEach` method is not the best choice here because it doesn't give you any control over the iteration. For example, you can't stop the iteration if you find an invalid person. Consider using a `for...of` loop instead. This gives you more control and makes your code easier to understand.